integration/maptest/map.go

```diff
@@ -172,8 +172,7 @@ func RunMapRevisionZero(ctx context.Context, t *testing.T, tadmin trillian.Trill
 		{
 			desc: "empty map has SMR at rev 0 but not rev 1",
 			hashStrategy: []hashStrategyAndRoot{
```

Comment:
This doesn't have to be a slice anymore. Add a TODO to simplify this test?

Reply:
Same argument applies to all the tests in this PR that are left with just one hash strategy.

Reply:
I'll remove the slice in the next PR which removes TEST_MAP_HASHER